docs/_source/guides/llms/practical_guides/collect_responses.md

```diff
@@ -81,33 +81,35 @@ If you feel that the disagreement between annotators is too high, especially for
-#### RatingQuestions
+Once you have unified your responses, you will have a dataset that's ready for [fine-tuning](fine_tune.ipynb). Remember to save your unified dataset following one of the methods explained in [Export a Feedback dataset](export_dataset.ipynb).
 
-* *Majority vote*: If a record has more than 2 submitted responses, you can take the most popular option as the final score. In the case of a tie, you can break it by choosing a random option or the lowest / highest score.
-* *Mean score*: For this technique, you can take all responses and calculate the mean score. That will be the final rating.
-* *Lowest / highest score*: Depending on how the question is formulated, you can take the `max` or `min` value. That will be the final rating.
+### Strategies
 
-#### LabelQuestions
+#### For labels: `LabelQuestion` and `MultiLabelQuestion`
 
-* *Majority vote (single-label questions)*: As with the rating, labels can be aggregated using the most popular option. For this option, we recommend that you have more than 2 submitted responses. In the case of a tie, you can break it by choosing a random option.
+* *Majority vote (single-label questions)*: Labels can be aggregated using the most popular option, for which you will need to have at least 3 submitted responses. In the case of a tie, you can break it by choosing a random option.
 * *Majority vote (multi-label questions)*: If you are aggregating labels from a multi-label question, it would be more correct to calculate the majority vote per label. That means that for each label you need to check whether the majority of the annotators selected that specific label or not.
 * *Weighted majority vote*: You may decide to give some of your annotators more weight than others when aggregating labels so that their decisions count more than others. Some reasons to consider a weighted majority might be: because some annotators tend to have better agreement with ground truth annotations, they are experts or they represent the demographic target for specific questions. If you want to choose this option, first calculate a score from 0 to 1 for each annotator, then apply these weights to their responses. Finally, sum all the values and choose the option with the highest score.
 * *Train with disagreements*: If your labels are meant to solve highly subjective tasks, like sentiment analysis or abusive language detection, you may want to consider other options that preserve the natural disagreement between annotators during training. This is specially helpful to avoid diluting the feedback of minorities within your annotation team. If you want to learn more about this approach, we recommend checking the different methods discussed in [Davani et al. (2021)](https://arxiv.org/pdf/2110.05719.pdf).
+
+#### For numerical values: `RankingQuestion` and `RatingQuestion`
```

Comment:
I think @davidberenstein1957 mentioned that the support for `RankingQuestion` is not ready yet, but he'll work on it for the next release, maybe we can either skip it or just add a warning that support via Python client is not ready yet?

Reply:
There are other approaches in that section that we don't support in the Python client and there's a disclaimer before the code section saying that we only cover some, so I think it should be ok to keep it there.